server/src/db/catalog/metrics.rs

```diff
@@ -56,8 +62,24 @@ impl CatalogMetrics {
             &chunk_lock_tracker,
         );
 
+        let storage_gauge = self.metrics_domain.register_gauge_metric_with_labels(
+            "chunks_storage",
```

Comment:
```suggestion
            "chunks_storage_count",
```

IIRC a metric should always reflect it's unit to some degree.

Reply:
I addressed this in my review.